python/tank/commands/setup_project_params.py

```diff
@@ -884,7 +884,8 @@ def _create_git_descriptor(self, git_uri):
             self._sg,
             Descriptor.CONFIG,
             {"type": "git_branch", "path": git_uri, "branch": "master"},
-            resolve_latest=True
+            resolve_latest=True,
+            local_fallback_when_disconnected=False,
```

Comment:
classic project setup: by avoiding the local fallback when we are not connected (which is critical in bootstrap), we avoid the confusing error messages about local caches.